BaseApi.Tests/V1/Controllers/BaseControllerTests.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using BaseApi.V1.Controllers;
+using BaseApi.V1.Infrastructure;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Routing;
+using NUnit.Framework;
+
+namespace BaseApi.Tests.V1.Controllers
```

Comment:
The folder names need to be changed (both `BaseAPI` and `BaseAPI.Tests`)

Reply:
Happy to do so, but if possible we should merge the initial renaming PR. And then I can merge master from there to this branch which should resolve it.

Reply:
I forgot the renaming PR hasn't been merged, ignore this comment please

Reply:
No worries at all :) With 2 approvals, I took the liberty of merging the renaming PR to master and then into this branch. Renaming issues should be resolved by now.